static/js/compose_height.js

```diff
@@ -0,0 +1,52 @@
+import autosize from "autosize";
+import $ from "jquery";
+
+let is_full_size = false; // true or false
+
+function get_ele_height(selector) {
+    return Number.parseInt($(selector).css("height"), 10);
+}
+
+function get_expected_full_height() {
+    // This returns the full size of the compose box
+    // Panels can have different height and this calculates it accordignly
+    return window.innerHeight - 100 - get_ele_height("#message_view_header") - get_ele_height("#panels");
```

Comment:
Did we rename `#panels`?  If not, let's add a prep commit renaming that to `#navbar_alerts`.

Reply:
Outdated, squashing this.